pkg/cfn/template/iam.go

```diff
@@ -0,0 +1,39 @@
+package template
+
+// IAMPolicy represents a CloudFormation AWS::IAM::Policy resource
+type IAMPolicy struct {
+	PolicyName *Value `json:",omitempty"`
+
+	Roles          []*Value        `json:",omitempty"`
```

Comment:
The idea is that we only define the fields we care about and use types according to our usage, i.e. this one is always a reference, so it's of generic `Value` type.